### PR TITLE
#1766 Fixed ibm/plex errors in console

### LINKED ISSUE #1766: Fix @ibm/plex errors in console
Ref: 3698

After upgrading to Carbon 11, we're seeing a lot of errors in console. Fix these errors -

console-errors-ibm-plex

### PR DESCRIPTION
**Pull request by nmgokhale**
Fixes #1766 

 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

File: canvas_modules/harness/Gruntfile.js [hunk 1]
```diff
@@ -61,6 +61,29 @@ module.exports = function(grunt) {
...
 			},
+			fonts: {
+				files: [{
+					expand: true,
+					flatten: false,
+					cwd: "./node_modules/@ibm/plex",
+					src: ["IBM-Plex*/**"],
+					dest: ".build/fonts"
+				},
+				{
```
Comment (matthoward366) [inline canvas_modules/harness/Gruntfile.js:72]: I'm a little surprised we have to flatten these. Is this what the carbon team suggests?

Comment (nmgokhale) [reply canvas_modules/harness/Gruntfile.js:72]: Errors are fixed even after setting `flatten: false`. Updated PR with this change. 
Carbon team only suggested to use `$font-path` with  `@carbon/react` package -
```
@use "@carbon/react" as * with (
	$font-path: "/fonts"
);
```

Comment (matthoward366) [reply canvas_modules/harness/Gruntfile.js:72]: I'm confused why we need node_modules/@ibm/plex/IBM-Plex-Sans/fonts/split?  According to the docs we shouldn't have to modify anything when copying over.

**Review by matthoward366 — APPROVED** [review]
Looks good

**Pull request merged**